src/main/java/mate/academy/springboot/datajpa/repository/CategoryRepository.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springboot.datajpa.repository;
+
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface CategoryRepository extends JpaRepository<Category, Long> {
+    @Query("UPDATE Category c SET c.name = ?2 WHERE c.id = ?1")
```

Comment:
why you do not use the default update?

Reply:
?

Reply:
use a method from JpaRepository update(entity)